src/main/java/org/folio/inventory/resources/Items.java

```diff
@@ -114,6 +121,22 @@ public void register(Router router) {
 
   }
 
+  private CompletableFuture<Void> markItemAsTargetStatus(
+    RoutingContext routingContext, WebContext webContext, Clients clients) {
+    Optional<ItemStatusName> itemStatusName = ItemStatusURL.getItemStatusNameForUrl(routingContext.request().uri());
```

Comment:
I find it easier to distinguish the first line of a method after a multi-line signature if there is a whitespace gap between them

```suggestion
  private CompletableFuture<Void> markItemAsTargetStatus(RoutingContext routingContext,
    WebContext webContext, Clients clients) {
    
    final var itemStatusName = ItemStatusURL.getItemStatusNameForUrl(routingContext.request().uri());
```